cli/asc.js

```diff
@@ -271,23 +274,25 @@ exports.main = function main(argv, options, callback) {
 
   // Set up base directory
   const baseDir = args.baseDir ? path.resolve(args.baseDir) : ".";
```

Comment:
```suggestion
  const baseDir = args.baseDir ? path.resolve(args.baseDir) : process.cwd();
```
This fixed the issue for me.  However, I'm not sure if this will work in the browser context.

Reply:
```suggestion
  const baseDir = path.resolve(args.baseDir || ".");
```
A simpler solution.

Reply:
Interesting find. Looks like the Windows code here https://github.com/AssemblyScript/assemblyscript/blob/b0e15ce80c127c4b25112cda183daf54b73d7cbd/cli/asc.js#L576-L580 is designed to skip the driver letter, but even does this if the path is not absolute. Changing to a `path.resolve` instead of a `path.join` in the first line has the same effect as giving an absolute `baseDir`.